src/book.py

```diff
@@ -129,12 +131,14 @@ def clean(self, lemmatize=True, deromanize=False, lemma_pos='v', inplace=False):
         inplace (default False): replaces the text of the object with cleaned text
         """
         cleaned = self._text
+        '
 
-        garbage = '\ufeff|â€™|â€"|â€œ|â€˜|â€\x9d|â€œi|_|â€'
+        garbage = '\ufeff|â€™|â€"|â€œ|â€˜|â€\x9d|â€œi|_|â€|£|éé|ô|à|â|ê|—£|éè|ü|é|œ|î|æ|ç|‘|é—|…|ö|è'
         cleaned = re.sub(garbage, '', cleaned)
         cleaned = cleaned.replace('-', ' ')
         cleaned = re.sub(r'\n+', ' ', cleaned)
         cleaned = cleaned.replace('-', ' ').replace('—', ' ')
+        cleaned = cleaned.replace('——', ' ').replace('——', ' ').replace('————', ' ')
```

Comment:
Aren't all of these just multiples of the previous line? i.e. replacing '-' with ' ' also replaces '--' with '  '

Reply:
good call. I've removed them.